script/formatting/formatter.py

```diff
@@ -34,15 +35,14 @@
 DEFAULT_DIRS.append(PELOTON_SRC_DIR)
 DEFAULT_DIRS.append(PELOTON_TESTS_DIR)
 
-CLANG_FORMAT = None
 CLANG_FORMAT_FILE = os.path.join(PELOTON_DIR, ".clang-format")
 
 ## ==============================================
 ##             HEADER CONFIGURATION
 ## ==============================================
 
 #header framework, dynamic information will be added inside function
```

Comment:
What should we do to format this section correctly? I was thinking of merging these individual strings to a complete whole. This will eliminate all these unwanted global variables as well.